### PR TITLE
Update figure 1 to show model drawing and resolved model.

### PR DESCRIPTION
**Pull request by grahamgower**
Maybe something like this for #15? I made the text `\scriptsize` and it still overflows the box. So not ideal. Thoughts?
fig1

### CODE REVIEW

File: models/IM-resolved.yaml [hunk 1]
```diff
@@ -0,0 +1,54 @@
+description: "Two-deme isolation-with-migration model."
+time_units: generations
+demes:
+- name: A
+  description: "The ancestral deme"
+  start_time: .inf
```
Comment (jeromekelleher) [inline models/IM-resolved.yaml:6]: This should be "Infinity" I think?

**Review by jeromekelleher — APPROVED** [review]
LGTM, but I'm not sure there's much point in trying to typeset the figure inline within the document. In the long-term, it won't survive final typesetting in the target journal and we will have to redo it. I guess we can either make a standalone PDF using latex which we include here, or someone does something in some drawing package.

File: models/IM-resolved.yaml (hunk 1 above)
Comment (grahamgower) [reply models/IM-resolved.yaml:6]: Only for the JSON output. YAML `.inf` is well defined.

**Comment by grahamgower** [comment]
> LGTM, but I'm not sure there's much point in trying to typeset the figure inline within the document. In the long-term, it won't survive final typesetting in the target journal and we will have to redo it. I guess we can either make a standalone PDF using latex which we include here, or someone does something in some drawing package.

I can push this into a `standalone` tex file, if you'd prefer? I'm pretty terrible with drawing programs, but happy to leave it to others if someone else wants to do that.

**Comment by jeromekelleher** [comment]
Sounds good. Let's see what others think. I'm happy to merge this for now and start writing around it.

**Comment by grahamgower** [comment]
Ok, I'll leave the standalone figure for later.

**Comment by molpopgen** [comment]
I like the image.  Using a standalone should be fine.  A Makefile can handle the details re: graphics formats for the journal.

**Pull request merged**